src/components/FileList.jsx

```diff
@@ -9,7 +7,7 @@ import AddFolder from '../containers/AddFolder'
 const FileList = ({
   t, f, context, files, error, showSelection, onFileEdit, onFolderOpen, onFileOpen, onFileToggle, onShowActionMenu
 }) => (
-  <div className={styles['fil-content-body']}>
+  <div>
```

Comment:
I know you have to wrap the render in a single element, but given the hard ass CSS architecture, I wonder if everything still works fine. I gotta test this to be sure.